kie-server-parent/kie-server-services/kie-server-services-prometheus/src/main/java/org/kie/server/services/prometheus/PrometheusJobListener.java

```diff
@@ -87,6 +101,13 @@ public void afterJobExecuted(AsynchronousJobEvent event) {
         final RequestInfo job = event.getJob();
         numberOfJobsExecuted.labels(defaultString(job.getDeploymentId()), String.valueOf(event.failed()), job.getCommandName()).inc();
         numberOfRunningJobs.labels(defaultString(job.getDeploymentId()), job.getCommandName()).dec();
+        if(job.getRetries() > 0 && job.getStatus().equals(STATUS.RETRYING)) {
```

Comment:
Notice that job "retries" is decremented before entering here. So, for example, if you add a test with 
`data.put("retries", 1);`
it will retry the command, but however, it will pass the first time with
```
retries=0
status=RETRYING
```
So that time, it won't be registered.